gpu_direct_rdma_access.c

```diff
@@ -69,13 +69,16 @@ int debug_fast_path = 0;
 #define COMP_ARRAY_SIZE 16
 #define TC_PRIO         3
 
+#define ZERO_BYTE_MSG_WR_ID UINT64_MAX  
```

Comment:
instead of ZERO_BYTE_MSG_WR_ID I suggest calling this #define WR_ID_FLUSH_MARKER. makes it very clear to understand what is it used for.